public/alcom/index.hbs

```diff
@@ -0,0 +1,139 @@
+{{#> page
+    title="ALCOM"
+    icon="/resources/alcom.svg"
+    with="social, before-header, scripts"
+}}
+
+{{#*inline "social"}}
+<meta property="og:title" content="ALCOM" />
+<meta property="og:type" content="website" />
+<meta property="og:url" content="https://vrc-get.anatawa12.com/{{locale.code}}/alcom" />
+<meta property="og:image" content="https://vrc-get.anatawa12.com/resources/alcom.svg" />
+<meta property="og:site_name" content="ALCOM" />
+<meta property="og:description" content="{{home.alcomDescription}}" />
+{{/inline}}
+
+{{#*inline "before-header"}}
+<div class="p-notification--negative u-no-margin--bottom" id="err-failedToLoadLatestVersion" hidden>
+    <div class="p-notification__content">
+        <h5 class="p-notification__title">{{common.error}}</h5>
+        <p class="p-notification__message">{{errors.failedToLoadLatestVersion}}</p>
+    </div>
+</div>
+{{/inline}}
+
+{{#*inline "scripts"}}
+<script src="/resources/js/alcom-versions-loader.js" defer></script>
+{{/inline}}
+
+{{#*inline "content"}}<div class="p-media-object--large">
+    <img src="/resources/alcom.svg" class="p-media-object__image" alt="">
+    <div class="p-media-object__details">
+        <h1 class="p-media-object__title">ALCOM</h1>
+        <p class="p-media-object__content">{{alcom.hero}}</p>
+        <ul class="p-media-object__meta-list">
+        </ul>
+    </div>
+</div>
+
+<a disabled="" class="p-button--positive is-processing has-icon" id="btn-download-loading">
+    <i class="p-icon--spinner u-animation--spin is-light"></i>
+    <span>{{common.loading}}</span>
+</a>
+
+<button disabled="" class="p-button--negative has-icon" id="btn-download-failed" hidden>
+    <i class="p-icon--close is-light"></i>
+    <span>{{common.error}}</span>
+</button>
+
+<div class="p-segmented-control inline-block max-width--xs" id="btn-download-main-container">
+    <div class="p-segmented-control__list">
+        <button onclick="window.location.href = _mainButtonLink"
+            class="p-segmented-control__button p-button--positive has-icon p-tooltip max-width--xs" id="btn-download-main"
+            hidden>
+            <i class="p-icon is-light" id="btn-download-main-platform"></i>
+            <span id="btn-download-main-text">{{common.download}} <code id="btn-download-main-version"
+                    class="no-background">0.0.0</code></span>
+            <span class="p-tooltip__message" role="tooltip" id="unsupported-platform-tooltip">This software
+                is not supported on your platform.</span>
```

Comment:
I think it's better to localize.

Reply:
Sorry, I missed that one.
Luckily, it was unused **:P**